src/test/regress/multi_mx_schedule

```diff
@@ -39,7 +39,8 @@ test: multi_mx_metadata
 test: master_evaluation master_evaluation_modify master_evaluation_select
 test: multi_mx_call
 test: multi_mx_function_call_delegation
-test: multi_mx_modifications local_shard_execution local_shard_copy
+test: multi_mx_modifications local_shard_execution
+test: local_shard_copy
```

Comment:
can't we execute this concurrent with `locally_execute_intermediate_results ` or such?

Reply:
Possibly, but it seemed safest to run it without any other test as it replicates the reference tables to coordinator. 

I guess if we run it concurrently with locally_execute_intermediate_results the test time wont be that different, is there any other reason you want that?